engine/src/main/java/com/datatorrent/stram/StreamingContainerManager.java

```diff
@@ -2246,6 +2306,50 @@ public LogicalOperatorInfo getLogicalOperatorInfo(String operatorName)
     return fillLogicalOperatorInfo(operatorMeta);
   }
 
+  public LogicalModuleInfo getLogicalModuleInfo(ModuleMeta module, boolean flatten)
+  {
+    LogicalModuleInfo lmi = new LogicalModuleInfo();
+    lmi.name = module.getName();
+    lmi.className = module.getModule().getClass().getName();
+    if (flatten) {
+      for (OperatorMeta operatorMeta : module.getDag().getAllOperators()) {
+        if (operatorMeta.getModuleName() == null) {
+          String fullPath = module.getFullName() + LogicalPlan.MODULE_NAMESPACE_SEPARATOR + operatorMeta.getName();
+          lmi.operators.add(fillLogicalOperatorInfo(getLogicalPlan().getOperatorMeta(fullPath)));
+        }
+      }
+      for (StreamMeta streamMeta : module.getDag().getAllStreams()) {
```

Comment:
@tweise lack of test case is because there are no test cases for rest api. I can add more test cases for REST api as a separate task.  Also the above code will include streams which are not part of module i.e stream which are connecting operators/modules outside of modules to the operators within the module. sub-dag returned in this will contain some streams whose one end is in the sub-dag and other end is not in the sub-dag.